src/main-app.component.html

```diff
@@ -0,0 +1,6 @@
+<side-menu></side-menu>
+<div id="main-content" class="split split-horizontal container-fluid main-content">
+    <key-map></key-map>
+    <legacy [link]="'keymapLegacy.html'" [class]="'keymap-legacy--edit'"></legacy>
+    <legacy [link]="'macroLegacy.html'" [class]="'macro-legacy--edit'"></legacy>
```

Comment:
This are leftovers from previous implementation. This will be removed when I will implement router, so here will only be `<router-outlet></router-outlet>`